libs/domains/environment/src/lib/slices/environment.actions.ts

```diff
@@ -95,7 +95,7 @@ export const postEnvironmentActionsCancelDeployment = createAsyncThunk<
   }
 })
 
-export const deleteEnvironmentActionsCancelDeployment = createAsyncThunk<
+export const deleteEnvironmentActions = createAsyncThunk<
```

Comment:
Should be `deleteEnvironmentAction` ?

You need to remove the code to refresh deployments history is not necessary